setup.py

```diff
@@ -147,9 +147,7 @@ def get_extensions():
     )
 
     is_rocm_pytorch = False
-    TORCH_MAJOR = int(torch.__version__.split('.')[0])
-    TORCH_MINOR = int(torch.__version__.split('.')[1])
-    if TORCH_MAJOR > 1 or (TORCH_MAJOR == 1 and TORCH_MINOR >= 5):
+    if torch.__version__ >= "1.5":
```

Comment:
Thanks for the PR @ABD-01 , 

I believe the original point of https://github.com/pytorch/vision/issues/4296 was to rely on the TorchVersion object: https://github.com/pytorch/pytorch/blob/1022443168b5fad55bbd03d087abf574c9d2e9df/torch/torch_version.py#L13, instead of just reverting the changes introduced in https://github.com/pytorch/vision/pull/4285

Reply:
Got it. 
In this repository there's version comparison only once and that is in [setup.py](https://github.com/pytorch/vision/blob/main/setup.py).
So I guess, there's no other file that need changes.

Reply:
Does `torch.__version__` object support comparisons to tuples? If yes, it would look much less error-prone: `if torch.__version__ >= (1, 5)` - at least it would be clear for code reader that some magic in `torch.__version__` is happening and that it's not just a string.

There are also similar problems with torchvision itself: https://github.com/HoBeom/detr/blob/master/util/misc.py#L21

Reply:
According to [the documentation](https://github.com/pytorch/pytorch/blob/1022443168b5fad55bbd03d087abf574c9d2e9df/torch/torch_version.py#L22-L32), `TorchVersion` supports comparing to strings, tuples and `Version` objects. Maybe the safest / most robust [would be to use `Version`](https://github.com/pytorch/pytorch/blob/1022443168b5fad55bbd03d087abf574c9d2e9df/torch/torch_version.py#L6), but it is worth checking more carefully